src/content/docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started/introduction-new-relic-cordova.mdx

```diff
@@ -19,6 +19,12 @@ The New Relic Cordova plugin will:
 * Add post-build scripts for uploading iOS symbolication files.
 * Upload Android Proguard mapping files.
 * Automatically instrument mobile applications built via Cordova.
+* Capture JavaScript errors
+* Distributed Tracing
```

Comment:
@ndesai-newrelic What can they do with distributed tracing? Are they tracking distributed traces?

Reply:
we are adding Distributed Tracing headers from Mobile and it will generate Distribute traces for Cordova apps.